.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ default_stages: [commit, push]
 
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
+    rev: v4.4.0
```

Comment:
This should be already in the main branch if you merged main!

Reply:
Sorry, this was an accident. Thanks for pointing it out!